ddl/ddl_db_change_test.go

```diff
@@ -292,14 +292,38 @@ type sqlWithErr struct {
 	expectErr error
 }
 
+type expectQuery struct {
+	sql  string
+	rows []string
+}
+
+// https://github.com/pingcap/tidb/pull/6249 fix the following two test cases.
+func (s *testStateChangeSuite) TestWriteOnlyWriteNULL(c *C) {
+	sqls := make([]sqlWithErr, 1)
+	sqls[0] = sqlWithErr{"insert t set c1 = 'c1_new', c3 = '2019-02-12', c4 = 8 on duplicate key update c1 = values(c1)", nil}
+	addColumnSQL := "alter table t add column c5 int not null default 1 after c4"
+	expectQuery := &expectQuery{"select c4, c5 from t", []string{"8 1"}}
+	s.runTestInSchemaState(c, model.StateWriteReorganization, "", addColumnSQL, sqls, expectQuery)
+}
+
+func (s *testStateChangeSuite) TestWriteOnlyOnDupUpdate(c *C) {
+	sqls := make([]sqlWithErr, 3)
+	sqls[0] = sqlWithErr{"delete from t", nil}
```

Comment:
Why do this? The `runTestInSchemaState` will drop table `t` to clean data.

Reply:
It is used to confirm the table is clean for this test.

Reply:
We done it  in `runTestInSchemaState`.

Reply:
emmm... in runTestInSchemaState, it inserts one row after create table. But in this case, it does not want to duplicate with any rows except in write only state. I'm afraid some one may add another row in runTestInSchemaState some day, it may be a duplicate row for this case. Then this case could pass directly, not work any more. So I add a delete here to confirm that it won't meet any other duplicate rows except the rows in this case.